opm/autodiff/FullyImplicitBlackoilSolver_impl.hpp

```diff
@@ -1897,22 +1900,61 @@ namespace {
                                                          std::array<double,MaxNumPhases>& B_avg,
                                                          int nc) const
     {
-        const Opm::PhaseUsage& pu = fluid_.phaseUsage();
         // Do the global reductions
-        for ( int idx=0; idx<MaxNumPhases; ++idx )
+#if HAVE_MPI
+        if(linsolver_.parallelInformation().type()==typeid(ParallelISTLInformation))
         {
-            if (active_[idx]) {
-                B_avg[idx] = B.col(idx).sum()/nc;
-                maxCoeff[idx]=tempV.col(idx).maxCoeff();
-                R_sum[idx] = R.col(idx).sum();
+            const ParallelISTLInformation& info =
+                boost::any_cast<const ParallelISTLInformation&>(linsolver_.parallelInformation());
+            // Compute the global number of cells and porevolume
+            std::vector<int> v(nc, 1);
+            auto nc_and_pv = std::tuple<int, double>(0,.0);
```

Comment:
I had to look twice at that initialization...I would prefer `(0, 0.0)` for readability (space after comma, not eliding the leading zero).